### ISSUE TITLE
Merge the gh-pages with master/dev branch

### ISSUE THREAD

**Issue by nschonni** [BrentOzarULTD/SQL-Server-First-Responder-Kit#436]
You can now put the jekyll page stuff in the code branchs so all the work can be done together https://github.com/blog/2228-simpler-github-pages-publishing

EX: If you hook up the actual publishing to the master, you can still do the work in DEV, and it will get deployed once DEV is merged into master

**Comment by BrentOzar** [comment]
Thanks! I need to burn the GH pages stuff with fire - it's a pain in the rear to edit, and I think I'm just going to point the firstresponderkit.org domain directly to the Github project instead, hahaha.
